perllib/FixMyStreet/Roles/CobrandEcho.pm

```diff
@@ -577,6 +577,8 @@ sub waste_munge_bulky_data {
             push @notes, $data->{"item_notes_$_"} || '';
             push @ids, $items{$item};
             push @photos, $data->{"item_photos_$_"} || '';
+            my $two = sprintf("%02d", $_);
+            $data->{"extra_ITEM_$two"} = $data->{"item_$_"};
```

Comment:
Whilst I see the reason for doing this, it's Bartec fields that is calling them ITEM_01 ITEM_02 etc, and I'm not sure we should base our storage around that. I also don't really want to clutter up the extra "fields" data with stuff that isn't for the backend, if that makes sense.
In 0288ed0292ea97ad13677c676c3a5a9ec4102b22 - for the waste_reconstruct_bulky_data function I got the list back from the IDs in Bulky_Collection_Bulky_Items - which is also not ideal because if they change the list of things, that wouldn't be able to display either.

So perhaps thinking about it we need a combination of both; we should store the items along with the report, for reference, but separately from backend data. So I guess I'd store it in the report's extra metadata (not fields) - I see in Waste.pm we store item_photo and location_photo as extra_metadata, perhaps we can store the list of items here as well. Perhaps not for Peterborough as it'd be duplicate data, but perhaps easier to do so? And then your bulky nice item list could use the relevant location (and my bulky reconstruct data also).